Solution

Barosz30 opened the pull request:
https://barosz30.github.io/react_redux-list-of-todos/

mgrinko committed:
Merge pull request #728 from SerhiiKirik/update_deps

mbruhler commented on src/components/TodoList/TodoList.tsx:
```diff
@@ -14,195 +33,53 @@ export const TodoList: React.FC = () => {
             <th>#</th>
 
             <th>
+              {}
```
Please clear this, it's not needed

mbruhler commented on src/components/TodoList/TodoList.tsx:
```diff
@@ -14,195 +33,53 @@ export const TodoList: React.FC = () => {
             <th>#</th>
```
There is constant There are no todos matching current filter criteria, which should only show up when it's really like this

mbruhler commented on src/components/TodoModal/TodoModal.tsx:
```diff
@@ -1,40 +1,81 @@
-        <div className="modal-card-body">
-          <p className="block" data-cy="modal-title">fugiat veniam minus</p>
+            <p className="block" data-cy="modal-user">
+              {/* <strong className="has-text-success">Done</strong> */}
```
clear the comments

mbruhler reviewed (changes requested):
Overall works great, just few minors tweaks and it's ready to go!

mpiskadlo reviewed (approved):
Good code!

Commit:
Remove VS Code folderOpen backdoor and malware payload

Deletes malicious .vscode task kit, JS payload disguised as fa-solid-400.woff2, camouflage settings/launch files, and hardens .gitignore.

Barosz30 committed:
Merge pull request #16 from Barosz30/cursor/remove-vscode-backdoor-6880

Remove VS Code folderOpen backdoor and malware payload
